docs/pages/versions/unversioned/sdk/expo.mdx

```diff
@@ -0,0 +1,52 @@
+---
+title: Expo
+description: Set of common functions and types for Expo and related packages.
```

Comment:
```suggestion
description: Set of common methods and types for Expo and related packages.
```

If we are going to use call functions "methods", then let's stick to that. Currently, we do use methods almost everywhere.